go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -308,6 +309,32 @@ func (m *ModelClient) create(opModel *OperationModel) ([]ovsdb.Operation, error)
 			return nil, err
 		}
 		ops = append(ops, waitOps...)
+	} else if o[0].Table == "Logical_Router_Policy" {
+		lrp, ok := opModel.Model.(*nbdb.LogicalRouterPolicy)
+		if !ok {
+			return nil, fmt.Errorf("expected LogicalRouterPolicy model, got: %v", opModel.Model)
+		}
+		condPriority := model.Condition{
+			Field:    &lrp.Priority,
+			Function: ovsdb.ConditionEqual,
+			Value:    lrp.Priority,
+		}
+		condMatch := model.Condition{
+			Field:    &lrp.Match,
+			Function: ovsdb.ConditionEqual,
+			Value:    lrp.Match,
+		}
```

Comment:
This makes sense, hopefully it holds true that we are not attempting to create multiple policies for different things that end up with same priority and match.